### DISCUSSION TITLE
爱莫能助 - 论狠心的码农是如何炼成的 | /cn/2018/09/unable-to-help/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1213] [Comments]
https://yihui.org/cn/2018/09/unable-to-help/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
肯定会有改善，如果提问者需要付出代价，那么他应该会更用心一些。这一点等我写《深度工作》总结的时候会再谈。以前我试验过一次就放弃了：<https://yihui.name/cn/2015/10/software-nanny/>

> ——原帖发布于 2018-09-25 13:54:15

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
有的小伙伴就是粘性特别大，你帮过他一次就成了他的救世主，下次碰到问题翻山越岭漂洋过海也要来找你。

> ——原帖发布于 2018-09-27 13:25:22

**Reply by coderkami1** [reply]
好有趣的形容，透着一股无奈

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
现在我们已经算是熟人了，所以你可以放心给我发邮件。

> ——原帖发布于 2018-10-01 15:05:53

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
不用惭愧。我都已经不记得这是哪年的事了。私事可以私下联系。

> ——原帖发布于 2018-10-09 01:58:39

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
来，特许你一个机会问，要问得不好的话，罚你去论坛回答别人的问题。

> ——原帖发布于 2018-10-12 02:45:13

[a comment by an automated account is not shown]
